### DISCUSSION TITLE
setting column value to NULL

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#6161] [Questions] [answered]
As title says, how? From dashboard.

![image](https://user-images.githubusercontent.com/56837461/160794766-87388198-17a7-48e4-b3b9-3dca42303a4e.png)

If i leave it empty, then value is "" string.
If i type null or NULL then is "null" string.

![image](https://user-images.githubusercontent.com/56837461/160795685-1c96a323-75dc-4c7a-a8c7-868494e61d79.png)

How to set to null value?
Left is string  NULL right is null as value NULL which is by default.

**Comment by GaryAustin1** [comment, marked as answer]
Does not look possible from the table editor.  
In the code it converts no value to empty string and everything else as string. 

function onChange(_value: string | undefined) {
    if (!_value) setValue('');
    else setValue(_value);
  }

Unless there is some undocumented way to enter an escape code in the editor, there is no other code to deal with setting to Null. 
Probably would be a good enhancement/bug request.

**Comment by jtkeyva** [comment]
@EdgarsLv @GaryAustin1 I found a workaround. You can't do it in the spreadsheet editor, but in the full row editor.

I noticed if you right click the cell and go to Edit Row it brings up the full editor form and when you have no content in the field there and click save, it saves it as NULL

**Reply by james-william-r** [reply]
@jtkeyva Nice catch! 👏
